src/shell/token.zig

```diff
@@ -0,0 +1,79 @@
+const ft = @import("../ft/ft.zig");
+const tty = @import("../tty/tty.zig");
+
+const max_tokens = 256;
+
+pub const Tokens = struct {
+	tokens: [max_tokens:null]?[:0]u8 = undefined,
+	len: usize = 0,
+};
```

Comment:
I think you could just define Tokens as a slice of a global array, there is no need for it to be null terminated and len is redundant

Reply:
Done in 9e00b5d